docs/delete-documents.md

```diff
@@ -0,0 +1,21 @@
+# Delete Documents
+
+If your API account has been granted delete access to a Document Type, the following end-point can be used to delete a document assigned to that Document Type.
+
+Note that this operation performs a "soft-delete", the document will still be in DocFinity but no longer available for viewing nor included in search results. Periodically, DocFinity will completely purge deleted documents.
+
+`POST https://{host}/docfinity/webservices/rest/document/delete`
```

Comment:
I think that the convention is to use "DELETE" instead of "POST".
DELETE https://{host}/docfinity/webservices/rest/document/{docid}

Reply:
I agree, but this is not our API, this is DocFinity's REST API.